tests/e2e/etcd_grpcproxy_test.go

```diff
@@ -90,6 +90,44 @@ func TestGrpcProxyAutoSync(t *testing.T) {
 	assert.Equal(t, []testutils.KV{{Key: "k1", Val: "v1"}}, kvs)
 }
 
+func TestGrpcProxyTlsVersions(t *testing.T) {
```

Comment:
Minor nit: this should be named `TestGrpcProxyTLSVersions`

```suggestion
func TestGrpcProxyTLSVersions(t *testing.T) {
```

Reply:
Agreed. But unfortunately it isn't detected by the linters.

Reply:
The linter complained precisely about this :rofl:

Reply:
eg check out this diff https://github.com/etcd-io/etcd/compare/98eb3ab908bd85d13330d42180945c9f6db22a79..3149bcf0c9923ac443f0cea47766afe552dd47a9

Reply:
scratch my previous comments, the linter doesn't seem to run on the e2e folder?

Reply:
The `tests` is already in the list. Anyway, could anyone raise a followup task and get it sorted out? thx

https://github.com/etcd-io/etcd/blob/12b47c2b935b6040a3874c512b2dd7f89dc0b599/scripts/test_lib.sh#L169

Reply:
fixed the naming manually now. I'm not able to even run the linter locally without OOMing my 64gig machine LOL

```
[tjungblu ~/git/etcd/tests]$ golangci-lint run --config /home/tjungblu/git/etcd/tools/.golangci.yaml
Killed
```

Reply:
Sorry, I was checking this and forgot to finish my reply. I'm unsure why `golangci-lint` is getting OOMKilled in your machine. It works for me. However, for some reason, it does not pick up the names of the functions inside test files. I'll follow up this on an issue soon.

Reply:
Raised https://github.com/mgechev/revive/issues/1124. The issue seems to be in the `revive` linter.

Reply:
Heads up: The linter won't check test file functions, as mentioned in https://github.com/mgechev/revive/issues/1124#issuecomment-2482055102, and there doesn't seem an interest by the maintainers to support them. I raised PR https://github.com/mgechev/revive/pull/1134 to document this behavior (already merged).